### PR TITLE
Hosted launch: Overview and architecture updates

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/2202

### CODE REVIEW

File: astro/hybrid-overview.md [hunk 1]
```diff
@@ -1,7 +1,24 @@
...
+Astro Hybrid is a self-hosted, Astronomer-managed service for data orchestration that is built for the cloud and powered by Apache Airflow. Astro Hybrid is built with a multi-plane architecture which consists of:
+
+- A control plane that is hosted by Astronomer.
+- A data plane that runs in your cloud.
+
+Astro Hybrid can be activated on Amazon Web Services (AWS), Google Cloud Platform (GCP), or Microsoft Azure. Astronomer runs Astro on managed Kubernetes services for each cloud provider, including AWS Elastic Kubernetes Service (EKS), Google Kubernetes Engine (GKE), and Azure Kubernetes Service (AKS).
+
+The control plane and data plane are managed by Astronomer and require no operational oversight by your organization. The Hybrid architecture ensures that tasks are executed securely within your corporate network.
+
+![Astro Hybrid architecture overview](/img/docs/hybrid-architecture-overview.png)
```
Comment (jwitz) [inline astro/hybrid-overview.md:16]:
```suggestion
![Astro Hybrid architecture overview](/img/docs/hybrid-architecture-overview.png)
```

File: astro/features.md [hunk 2]
```diff
@@ -5,18 +5,13 @@ id: features
...
-- The ability to run multiple clusters in your organization's network on AWS, GCP, or Azure.
 - Astro Runtime, a collection of Docker images that provides a differentiated data orchestration experience. See [Runtime image architecture](runtime-image-architecture.md).
 - Timely support for the latest major, minor, and patch versions of the Apache Airflow open source project.
 - Support for role-based access control (RBAC) and single sign-on (SSO) for secure user management and authentication. See [Configure an identity provider (IdP)](configure-idp.md) and [User permissions](user-permissions.md).
 - An observability experience that gives you data lineage across your ecosystem as well as real-time insight into the health and resource consumption of your data pipelines on Astro. See [Data lineage](data-lineage.md) and [Deployment metrics](deployment-metrics.md).
+- - A suite of automation tools for programmatically managing Airflow environments across teams and development cycles. See [CI/CD](ci-cd-templates/template-overview.md) and [Manage Deployments as code](manage-deployments-as-code.md).
```
Comment (jwitz) [inline astro/features.md:17]:
```suggestion
- A toolset for programmatically managing Airflow environments across teams and development cycles. See [CI/CD](ci-cd-templates/template-overview.md) and [Manage Deployments as code](manage-deployments-as-code.md).
```

File: astro/astro-architecture.md [hunk 3]
```diff
@@ -5,22 +5,17 @@ id: astro-architecture
...
+Astro Hosted is a managed service for data orchestration that is built for the cloud and powered by Apache Airflow. Your Airflow environments are hosted and managed entirely by Astronomer, enabling you to shift your focus from infrastructure to data. 
 
-- A control plane that is hosted by Astronomer.
-- A data plane that can run either in your cloud or in a single-tenant account hosted by Astronomer.
+Astro Hosted consists of three core components for managing Airflow, all hosted on Astronomer's cloud:
 
-Astro can be activated on Amazon Web Services (AWS), Google Cloud Platform (GCP), or Microsoft Azure. Astronomer runs Astro on managed Kubernetes services for each cloud provider, including AWS Elastic Kubernetes Service (EKS), Google Kubernetes Engine (GKE), and Azure Kubernetes Service (AKS).
+- The _Astro control plane_ is Astronomer's interface for managing your Airflow environments in the cloud. It includes the Cloud UI, the Astro CLI, and Cloud API.
+- _Astro clusters_ are dedicated Kubernetes clusters that can run multiple Airflow environments.
```
Comment (jwitz) [inline astro/astro-architecture.md:13]:
```suggestion
- An _Astro cluster_ is a dedicated Kubernetes cluster that can run multiple Airflow environments.
```

File: astro/astro-architecture.md [hunk 4]
```diff
@@ -5,22 +5,17 @@ id: astro-architecture
 description: Astro uses a multi-plane structure to help make Airflow more secure and reliable. Learn how the control plane and data plane work together on the cloud.
 ---
 
-Astro is a managed service for data orchestration that is built for the cloud and powered by Apache Airflow. Astro is built with a multi-plane architecture which consists of:
+Astro Hosted is a managed service for data orchestration that is built for the cloud and powered by Apache Airflow. Your Airflow environments are hosted and managed entirely by Astronomer, enabling you to shift your focus from infrastructure to data.
```
Comment (paolaperaza) [inline astro/astro-architecture.md:8]: I'd prob slightly re work this doc to:

- Say what Astro is (value)
- Then a section on how you can run it 
- Acknowledge default of Hosted but put it against Hybrid
- Still have those Hybrid docs links

Know the intent is to go in on Hosted, but Having a hard time imagining how we avoid this direct comparison w Hybrid? Eso if both on pricing page

**Review by paolaperaza — COMMENTED** [review]
Lemme give this another read tomorrow (Monday) morning

**Pull request merged**